test/test_cpp_api_parity.py

```diff
@@ -450,6 +504,12 @@ def _process_test_params(test_params_dict, module_metadata, device):
     else:
         raise RuntimeError("Unexpected input type: {}".format(type(example_inputs)))
 
+    # We set all inputs to torch.nn module to requires grad, so that the backward test can always be run.
```

Comment:
(no action items) You could also just turn requires_grad on if dtype is not integral, but I'm not sure that is sufficient and we can worry about that later when parity tests are added for the Embedding layers